QC8Monitor/StatusVmon904LV.py

```diff
@@ -0,0 +1,714 @@
+import cx_Oracle
```

Comment:
do we really need two different code for this.
its better to make 1 script that does both